src/amo/components/AddonCompatibilityError/index.js

```diff
@@ -95,8 +95,13 @@ export class AddonCompatibilityErrorBase extends React.Component {
       ), { downloadUrl });
     }
 
+    // Make the "you should download firefox" error message less scary than
+    // the rest of them: https://github.com/mozilla/addons-frontend/issues/4547
+    const noticeType = reason === INCOMPATIBLE_NOT_FIREFOX ?
+      'generic' : 'error';
```

Comment:
I guess it would be hard to switch this to a [download button](https://github.com/mozilla/addons-frontend/blob/62015810feb4797a5fc5019581e745bfd21e5420/src/amo/components/AddonCompatibilityError/index.js#L62)? The `Notice` component already supports it. If it's difficult then we should do it in a follow-up issue.

Call to actions look like this: https://design.firefox.com/photon/components/message-bars.html#call-to-action

Reply:
It supports a button but not one with a `href` prop, just an `onClick` action. It's worth doing separately I think if we want this in for this week. I thought it'd be quick but I think it's worth leaving for now.